app/src/main/java/com/example/cosc341_project/data_classes/SightingPost.java

```diff
@@ -0,0 +1,67 @@
+package com.example.cosc341_project.data_classes;
+
+import java.util.Arrays;
+
+/**
+ * <p>
+ *     Extends <code>Post</code>.
+ * </p>
+ * <p>
+ *     Includes attributes <code>imageName</code> and <code>location</code>, which have no
+ *     default values and must be set with setters. Also, currently they are both Strings,
```

Comment:
The image could probably be a "Bitmap" variable, unless we want to store the image locally and use the image path as a string. Bitmap would probably be easier.

Reply:
Sounds good. I'm not gonna update it for this branch because everything else is ready to merge, but will figure it out soon.